controllers/org_rbac_controller.go

```diff
@@ -52,14 +53,15 @@ func (r *OrganizationRBACReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, nil
 	}
 
+	var err error
```

Comment:
I personally find `var errs []error; multierr.Combine(errs...)` more readable but that is my opinion. I'm fine with both.